client/src/components/landingpage.js

```diff
@@ -0,0 +1,60 @@
+// landingpage
+
+import React, {useState, useEffect} from 'react';
+import axios from 'axios';
+
+function LandingPage() {
+
+	const [icon, chooseIcon] = useState('Anything');
+	const [input, updateInput] = useState("");
+	const [placeholder, setPlaceholder] = useState("Enter a City and State or a Zipcode")
+
+	useEffect(() => {
+		if (navigator.geolocation) {
+			navigator.geolocation.getCurrentPosition(displayLocationInfo)
+		}
+	})
+
+	function displayLocationInfo(position) {
+		const lng = position.coords.longitude;
+		const lat = position.coords.latitude;
+		const API_KEY = `${process.env.REACT_APP_GEO_CODER_API_KEY}`;
+		axios.get(`https://api.opencagedata.com/geocode/v1/json?q=${lat}+${lng}&key=${API_KEY}`)
+			.then((response) => {
+				let city = response.data.results[0].components.town
+				let state = response.data.results[0].components.state_code
+				setPlaceholder(`${city}, ${state}`)
+			}, (error) => {
+				console.log(error)
```

Comment:
We should probably do something to handle any errors.  Maybe set it in state and display something below the input?

Reply:
This is the same idea as above, we could display a message saying their location could not be found.